src/doc/DocAnnotator.js

```diff
@@ -774,15 +772,17 @@ class DocAnnotator extends Annotator {
      * Bail if mid highlight and click is outside highlight/selection
      *
      * @param {Event} event - Mouse event
-     * @return {void}
+     * @return {boolean} - Whether or not event was consumed
      */
-    resetHighlightOnOutsideClick(event: Event) {
+    resetHighlightOnOutsideClick(event: Event): boolean {
         const selection = window.getSelection();
         const isClickOutsideCreateDialog = this.isCreatingHighlight && !util.isInDialog(event);
         if (!docUtil.isValidSelection(selection) && isClickOutsideCreateDialog) {
             this.lastHighlightEvent = null;
             this.resetHighlightSelection(event);
+            return true;
```

Comment:
Previously nothing was reading this boolean and nothing in your change is reading the boolean. Can this go back to returning null?